test/storage/2019-07-07/azblob/service.go

```diff
@@ -57,7 +56,7 @@ func (client *serviceOperations) GetAccountInfo(ctx context.Context) (*ServiceGe
 }
 
 // getAccountInfoCreateRequest creates the GetAccountInfo request.
-func (client *serviceOperations) getAccountInfoCreateRequest(urlParameter string) (*azcore.Request, error) {
+func (client *serviceOperations) getAccountInfoCreateRequest() (*azcore.Request, error) {
 	urlPath := "/"
```

Comment:
Would be good to remove this, avoiding the call to `client.u.Parse()` thus removing creating a new `url.URL` (I'm fine with deferring it to a separate PR if that works better).